batch/batch/utils.py

```diff
@@ -45,6 +45,24 @@ async def wrapped(request):
     return wrapped
 
 
+def add_metadata_to_request(fun):
+    @wraps(fun)
+    async def wrapped(request, *args, **kwargs):
+        request['batch_telemetry'] = {'operation': fun.__name__}
+
+        batch_id = request.match_info.get('batch_id')
+        if batch_id is not None:
+            request['batch_id'] = batch_id
```

Comment:
I think this should be `request['batch_telemetry']['batch_id'] = batch_id`. (and same for job id below)